### ISSUE TITLE
when label_mat=train_p，Y_value=1，why use (Y_hat_values-1) to calculate the loss when the training data does not distinguish between positive and negative sets of data ?

### ISSUE THREAD

**Issue by zyyuan180** [luoyunan/DTINet#21]
a

**Comment by luoyunan** [comment]
Could you make it more clear and let us know which lines of code you were referring to? Thanks!
